### ISSUE TITLE
Gerar configs baseados nos nossos style guides

### ISSUE THREAD

**Issue by dvinciguerra** [bioritmo/style-guide#13]
Não poderíamos gerar configs para as ferramentas de verificação baseadas nos nossos style guides descritos aqui?

**Comment by shadowmaru** [comment]
Sim, só precisaria terminar de definir o style guide (por enquanto são esboços)
